compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_freebsd.h

```diff
@@ -726,6 +726,8 @@ struct __sanitizer_cpuset {
 
 typedef struct __sanitizer_cpuset __sanitizer_cpuset_t;
 extern unsigned struct_cpuset_sz;
+
+typedef unsigned long long __sanitizer_eventfd_t;
```

Comment:
if it's the same time as posix, can you have just a single typedef there in part enabled for freebsd as well?